### PR TITLE
MWPW-130723 | Fixes for Beta Launch Page Enhancements

### PR DESCRIPTION
**Pull request by apganapa**
Fixes MWPW-130723

Fixed below items,

- Make a project alignment changes to title and page list.
- Padding for areas in promotions.
- CTA button and making free-plan-widget static and removing animation from the place.

Test URLs:
- Before: https://stage--express-website--adobe.hlx.page/express/beta
- After: https://<branch>--express-website--adobe.hlx.page/express/](https://mwpw-130723--express-website--adobe.hlx.page/express/beta)

### CODE REVIEW

File: express/styles/styles.css [hunk 1]
```diff
@@ -516,6 +516,14 @@ main .section:last-of-type>.default-content-wrapper {
   padding-bottom: 40px;
 }
 
+main .section:first-of-type {
+  margin-bottom: -75px;
+}
```
Comment (qiyundai) [inline express/styles/styles.css:521]: This feels very risky to me. Can we check again to make sure to target only the desired use case?

File: express/styles/styles.css [hunk 2]
```diff
@@ -544,6 +552,19 @@ main .section:last-of-type>.default-content-wrapper {
   main p {
     font-size: var(--body-font-size-l);
   }
+
+  main .section:first-of-type {
+    margin-bottom: 0;
```
Comment (qiyundai) [inline express/styles/styles.css:557]: Feels risky here too. This affects all of the first sections across the entire site. Maybe I'm missing the picture. What's the ask behind this?

File: express/scripts/scripts.js [hunk 3]
```diff
@@ -2381,7 +2381,7 @@ function removeMetadata() {
 
 export async function buildStaticFreePlanWidget() {
   const placeholders = await fetchPlaceholders();
-  const widget = createTag('div', { class: 'free-plan-widget' });
+  const widget = createTag('div', { class: 'free-plan-widget', style: `display: block !important` });
```
Comment (qiyundai) [inline express/scripts/scripts.js:2384]: Can you check if we can avoid using !important here? I actually believe we can debug this from CSS without injecting inline styling.

**Review by qiyundai — CHANGES_REQUESTED** [review]
Looking pretty good. A few things to check. Thanks!

File: express/styles/styles.css [hunk 4]
```diff
@@ -544,6 +552,19 @@ main .section:last-of-type>.default-content-wrapper {
...
+  }
+
+  main div.section.promotion-container {
```
Comment (qiyundai) [inline express/styles/styles.css:560]: Is it possible that this should be a block level css change?

**Review by qiyundai — APPROVED** [review]
lgtm! Thanks!

**Review by JingleH — APPROVED** [review]

**Pull request merged**